log/log.go

```diff
@@ -25,9 +44,13 @@ type Logger struct {
 //==============================================================================
 
 // New returns a instance of a logger.
-func New(w io.Writer, levelHandler func() int) *Logger {
+func New(w io.Writer, levelHandler func() int, flags int) *Logger {
+	if flags == 0 {
```

Comment:
We can remove the check for 0 and ask the user to pass Ldefault. Zero would mean no flags.